File: app/src/main/java/com/tohsoft/app/receivers/BootUpReceiver.java
package com.tohsoft.app.receivers;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class BootUpReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        // Restart some processes when the phone has just finished booting
    }
}

File: app/src/main/java/com/tohsoft/app/receivers/BootUpReceiver.kt
package com.tohsoft.app.receivers

import android.annotation.SuppressLint
import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class BootUpReceiver : BroadcastReceiver() {

    @SuppressLint("UnsafeProtectedBroadcastReceiver")
    override fun onReceive(context: Context, intent: Intent) {
        // Restart some processes when the phone has just finished booting
    }
}